modules/nw-sriov-installing-operator.adoc

```diff
@@ -69,8 +69,25 @@ spec:
   sourceNamespace: openshift-marketplace
 EOF
 ----
+. Create an `SriovoperatorConfig` resource by entering the following command:
++
+[source,terminal]
+----
+$ cat <<EOF | oc create -f -
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovOperatorConfig
+metadata:
+  name: default
+  namespace: openshift-sriov-network-operator
+spec:
+  enableInjector: true
+  enableOperatorWebhook: true
+  logLevel: 2
+  disableDrain: false
+EOF
+----
 
-. To verify that the Operator is installed, enter the following command:
+. Verify that the Operator is installed by entering the following command:
```

Comment:
Mimic verification step as seen here: https://docs.openshift.com/container-platform/4.16/networking/ovn_kubernetes_network_provider/configuring-egress-traffic-for-vrf-loadbalancer-services.html#nw-egress-service-ovn_configuring-egress-traffic-loadbalancer-services (CTRL+F "verification")

So it would come out to something like:

```suggestion
.Verification

* Verify that the Operator is installed by entering the following command:
+
[source,terminal]
----

----
+
.Example output
+
[source,terminal]
----

----
```